rust/tests/behaviour/query/language/steps.rs

```diff
@@ -105,14 +105,16 @@ generic_step_impl! {
     #[step(expr = "get answers of typeql get")]
     pub async fn get_answers_typeql_get(context: &mut Context, step: &Step) -> TypeDBResult {
         let parsed = parse_query(step.docstring().unwrap())?;
-        context.answer = context.transaction().query().get(&parsed.to_string())?.try_collect::<Vec<_>>().await?;
+        let answer = context.transaction().query().get(&parsed.to_string())?.try_collect::<Vec<_>>().await?;
+        context.answer = answer;
```

Comment:
We should be able to write this on one line, but the borrow checker gets confused by the `.await`, and thinks that we attempt to borrow `context` twice at the same time.